src/main/java/cholog/supp/api/post/service/PostService.java

```diff
@@ -19,15 +29,67 @@ public class PostService {
 
     private final PostRepository postRepository;
     private final MemberStudyMapRepository memberStudyMapRepository;
-    private final StudyGroupRepository studyGroupRepository;
+    private final CommentRepository commentRepository;
 
-    public void createPost(Member member, CreatePostRequest createPostRequest) {
+    public Long createPost(Member member, CreatePostRequest createPostRequest) {
         MemberStudyMap memberStudy = memberStudyMapRepository.findByStudyGroupIdAndMemberId(
                 createPostRequest.studyId(), member.getId())
             .orElseThrow(() -> new IllegalArgumentException("존재하지 않는 스터디 입니다."));
         StudyGroup studyGroup = memberStudy.getStudyGroup();
-        postRepository.save(
+        Post post = postRepository.save(
             new Post(member, studyGroup, createPostRequest.title(),
                 createPostRequest.description()));
+        return post.getId();
+    }
+
+    @Transactional(readOnly = true)
+    public List<PostResponse> getPostList(PostsRequest postsRequest) {
+        List<Post> allPost = postRepository.findAllByStudyIdDesc(postsRequest.studyId());
+        return allPost.stream()
+            .map(it -> new PostResponse(it.getId(), it.getTitle(), it.getCreatedDate())).toList();
+    }
+
+    public void modifyPost(Member member, ModifyPostRequest modifyPostRequest) {
+        Post post = verifyPostOwner(member, modifyPostRequest.postId());
+        post.modifyPost(new ModifyPost(modifyPostRequest.title(), modifyPostRequest.description()));
+    }
+
+    public void deletePost(Member member, Long postId) {
+        verifyPostOwner(member, postId);
+        commentRepository.deleteAllByPostId(postId);
+        postRepository.deleteById(postId);
+    }
+
+    private Post verifyPostOwner(Member member, Long postId) {
+        Post post = postRepository.findById(postId)
+            .orElseThrow(() -> new IllegalArgumentException("존재하지 않는 질문글 입니다."));
+        if (!Validation.verifyMember(member, post.getMember().getId())) {
+            throw new IllegalArgumentException("잘못된 접근입니다.");
+        }
+        return post;
+    }
+
+    public EachPostResponse getEachPost(Member member, Long postId) {
+        Post post = postRepository.findById(postId)
+            .orElseThrow(() -> new IllegalArgumentException("존재하지 않는 질문글 입니다."));
+        boolean isMyPost = Validation.verifyMember(member, post.getMember().getId());
+        EachPost eachPost = new EachPost(post, isMyPost);
+        List<EachComment> comments = makeComments(post.getStudy().getId(),
+            commentRepository.findAllByPostId(postId),
+            member);
+        return new EachPostResponse(eachPost, comments);
+    }
+
+    private List<EachComment> makeComments(Long studyId, List<Comment> comments, Member member) {
```

Comment:
```suggestion
    private List<EachComment> getComments(Long studyId, List<Comment> comments, Member member) {
```
`makeComments`라는 메서드명은 의견을 생성하는것으로 보이니 가져온다는 표현이 맞지 않을까??

Reply:
그러네!!!!!!!

Reply:
`List`도 좋은데 내 생각으로는 `Comments` 자체가 복수형이라 없어도 괜찮을거 같어

Reply:
List랑 관련 없는 것들에 보통 복수형으로 써와서(List를 전혀 받지 않는 request DTO의 이름을 복수형으로 명시한다던지) 반환값이 List 인것은 붙여주고자했어!!
어띃게생각해?? 그래서 난 이거는 List를 붙여주는게 좋을거라구 생각했어

Reply:
수민이 너가 네이밍한 방식이 따로 존재하면 그 틀을 유지하면 좋을거 같아👍👍👍